src/deluge/model/song/song.cpp

```diff
@@ -5758,6 +5758,18 @@ ModelStackWithThreeMainThings* Song::addToModelStack(ModelStack* modelStack) {
 	return modelStack->addTimelineCounter(this)->addOtherTwoThingsButNoNoteRow(&globalEffectable, &paramManager);
 }
 
+// get's the model stack for Song level parameters that are being edited
+// used in performance view and in automation arranger view
```

Comment:
This documentation should move to `song.h` and use `///` comments so doxygen picks it up. I'd probably write something like this:

Gets a modelstack with the song-global unpatched param `paramID`.

That docstring tells me the most important information about the function:
  - What parameters does it accept (an `UNPATCHED_PARAM_*` paramID)
  - What, as the user of this function, can I expect to be true about its return value (or more generally, the state of the system now that I've called the function).